test/bootleg/tasks/deploy_task_functional_test.exs

```diff
@@ -40,4 +40,15 @@ defmodule Bootleg.Tasks.DeployTaskFunctionalTest do
       end)
     end)
   end
+
+  test "deploy/1 deploys the release to the target hosts from a remote release_workspace path", %{hosts: [host]} do
+    role :app, [host.ip], port: host.port, user: host.user, password: host.password,
+      silently_accept_hosts: true, workspace: "workspace", release_workspace: "/fixtures"
+
+    File.cd!("test/fixtures", fn ->
+      capture_io(fn ->
+        invoke :deploy
```

Comment:
same as above, please add an assertion to make sure that the deploy was successful. it doesn't have to be anything super complex, maybe just a quick check that one of the unpacked files exists where we would expect it to